arc_cache_test.go

```diff
@@ -259,3 +352,187 @@ func TestPutManyCaches(t *testing.T) {
 	trap("PunMany has hit datastore", cd, t)
```

Comment:
I know it's not actually in the diff, but I happen to see a typo here.

(And it's the best typo.)